all.json

```diff
@@ -11055,6 +11055,41 @@
 		"zinotiger.com",
 		"zipwallet.net",
 		"zonefix.net",
-		"zwalletconnect.com"
+		"zwalletconnect.com",
+		"wallreceivecondapp.click",
+		"wallletuserconect.company",
+		"validationport.com",
+		"webswalletconnect.com",
+		"mainconnectedwallet.com",
+		"integrals-dexs.net",
+		"ddns.us",
+		"bankk-11.cf",
+		"integral-dex-bridges.net",
+		"fastautolinked.com",
+		"lopezgarciaabogados.com",
+		"walldmconecttkprecieve.live",
+		"myhuaweicloud.com",
+		"integral-dexs.com",
+		"aliancrypto.com",
+		"portalsync.live",
+		"walletconnecti.com",
+		"apps-syncs.net",
+		"q2rv.com",
+		"defisolutionprotocol.online",
+		"appsregistry.io",
+		"dataconnectted.com",
+		"m26securiter.info",
+		"wlt-register.live",
+		"trustwallet-verifyid-16.ml",
+		"connectiwallets.com",
+		"wcsynchprotocols.com",
+		"mobilewalletsdapp.com",
+		"waconlinkks.click",
+		"ajudacef.com",
+		"naskizz.com",
+		"teachablei.com",
+		"integrals-dex.com",
+		"walletdappvalidator.io",
+		"pagekite.me"
```

Comment:
```suggestion
```

Also a known tunelling solution...

Reply:
And "myhuaweicloud.com" which redirects to "huaweicloud.com"

Reply:
Sorry, missed that one, long day.

Reply:
For this we should add the full url:
`syncrvalidation.pagekite.me`

The bucket from "myhuaweicloud.com":
`vnwerolivh-3w4erhgv-38ih49-ghwehgfv-3w4g.obs.na-mexico-1.myhuaweicloud.com`
doesn't exist anymore, so I think we can safely leave that entry out entirely.